listenbrainz/webserver/templates/index/index.html

```diff
@@ -22,7 +22,7 @@ <h2 class="page-title">Welcome to ListenBrainz!</h2>
         account, you can <a href="https://musicbrainz.org/register">create one</a>! Once you have an
         account, you can <a href="{{ url_for('profile.import_data') }}">import</a>
         data from Last.FM, <a href="{{ url_for('profile.import_data') }}">connect your Spotify account</a>,
```

Comment:
```suggestion
        data from Last.FM, <a href="{{ url_for('profile.connect_spotify') }}">connect your Spotify account</a>,
```

This is unrelated, but the spotify import link should go to the spotify page, now that the import page doesn't mention spotify.